### PR TITLE
Merge all SVG icons into a single file

### PR DESCRIPTION
**Pull request by mkitt**
Just prepping these for the incoming icons. This should be everything from the markup side of things. The CSS is still a mess, but we'll start to tackle that when we upgrade the icons and start moving the CSS over too.

Might be easier to go commit by commit for review.

[#140987175](https://www.pivotaltracker.com/story/show/140987175)

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Nice, more deletions than insertions :red_circle:</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/styles/cso.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ import { css } from 'glamor'
 import * as jso from './jso'
 
 // -------------------------------------
+// Helpers (Web specific)
+
+export const combine = (...styles: Array<string>) => styles.join(' ')
+
+// -------------------------------------
 // Configuration (Web specific)
```
Comment (mkitt) [inline src/styles/cso.js]: The commit notes from 0ebe119 should explain this one...

File: src/styles/cso.js [hunk 2]
```diff
@@ -55,8 +55,8 @@ export const fit = css(jso.fit)
 export const overflowXHidden = css(jso.overflowVisible)
 export const overflowHidden = css(jso.overflowHidden)
 export const overflowScroll = css({
-  '-webkit-overflow-scrolling': 'touch',
-  'overflow-y': 'auto',
+  WebkitOverflowScrolling: 'touch',
+  overflowY: 'auto',
 })
```
Comment (mkitt) [inline src/styles/cso.js]: Should fix those warnings. 🔨

File: src/styles/cso.js (hunk 1 above)
Comment (rynbyjn) [inline src/styles/cso.js]: Makes sense

File: src/styles/cso.js (hunk 2 above)
Comment (rynbyjn) [inline src/styles/cso.js]: Nice

**Pull request merged**